op-node/p2p/gossip.go

```diff
@@ -165,7 +174,7 @@ func BuildGlobalGossipParams(cfg *rollup.Config) pubsub.GossipSubParams {
 	params.Dlazy = DefaultMeshDlazy            // gossip target
 	params.HeartbeatInterval = gossipHeartbeat // interval of heartbeat
 	params.FanoutTTL = 24 * time.Second        // ttl for fanout maps for topics we are not subscribed to but have published to
-	params.HistoryLength = 12                  // number of windows to retain full messages in cache for IWANT responses
+	params.HistoryLength = 5                   // number of windows to retain full messages in cache for IWANT responses
```

Comment:
Is this to reduce the memory usage?

Reply:
yes, it was somehow taking up quite a bit of memory, but i think we can change it back....we had an OOMKill issue in the pods, because we have more topics now, not sure if this was the fix or another thing, but will take a look.